lib/trento/application/projectors/cluster_projector.ex

```diff
@@ -63,8 +63,10 @@ defmodule Trento.ClusterProjector do
       deregistered_at: deregistered_at
     },
     fn multi ->
+      cluster = Repo.get!(ClusterReadModel, cluster_id)
```

Comment:
Just as a comment, in other `project` functions, we have this pipelined:
```
changeset =
    ClusterReadModel
    |> Repo.get!(cluster_id)
    |> ClusterReadModel.changeset(%{
         deregistered_at: deregistered_at
    })
```
Just in case if you prefer it :stuck_out_tongue_closed_eyes:

Reply:
You triggered my ocd 👁️ 

Everything refactored to this style